amivapi/users/model.py

```diff
@@ -96,6 +96,7 @@
             # Fields the user can modify himself
             'password': {
                 'type': 'string',
+                'minlength': 7,
```

Comment:
Any particular reason why it's 7?

Reply:
It was the proposal from the linked issue. It does not necessarily need to be 7, but I think this reasonable.

Reply:
Fair enough!